### PR TITLE
Filter non-paid team's machine from kd operations

### PR DESCRIPTION
**Pull request by cihangir**

### CODE REVIEW

File: go/src/koding/kites/kontrol/kontrol/handlers_test.go [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+				activeGroup2.Id.Hex(): []*KodingKiteWithToken{kite2},
+			},
+			response: &GetKodingKitesResult{
+				Kites: []*KodingKiteWithToken{kite1, kite2},
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
```
Comment (ppknap) [inline go/src/koding/kites/kontrol/kontrol/handlers_test.go]: :zap: 1.7 is here :)

**Review by ppknap — APPROVED** [review]

File: go/src/koding/db/mongodb/modelhelper/group.go [hunk 2]
```diff
@@ -21,6 +21,17 @@ func GetGroupById(id string) (*models.Group, error) {
 	return group, Mongo.Run(GroupsCollectionName, query)
 }
 
+// GetGroupsByIds returns groups by their given IDs
+func GetGroupsByIds(ids ...bson.ObjectId) ([]*models.Group, error) {
+	groups := make([]*models.Group, 0)
```
Comment (rjeczalik) [inline go/src/koding/db/mongodb/modelhelper/group.go]: nitpick: can be just `var groups []*models.Group`

https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices

File: go/src/koding/kites/kontrol/kontrol/handlers_test.go [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+		response *GetKodingKitesResult
+	}{
+		{
+			name: "should stay same - everything is perfect",
+			getKodingKitesResult: &GetKodingKitesResult{
+				Kites: []*KodingKiteWithToken{kite1, kite2},
+			},
+			groups: []*models.Group{activeGroup1, activeGroup2},
+			kitesByGroupID: map[string][]*KodingKiteWithToken{
+				activeGroup1.Id.Hex(): []*KodingKiteWithToken{kite1, kite2},
```
Comment (rjeczalik) [inline go/src/koding/kites/kontrol/kontrol/handlers_test.go]: `go fmt -s -w . ` could simplify this to:

```
activeGroup1.Id.Hex(): {kite1, kite2},
```

File: go/src/koding/db/mongodb/modelhelper/group.go (hunk 2 above)
Comment (cihangir) [inline go/src/koding/db/mongodb/modelhelper/group.go]: i just followed the _(un)structure_ under go/src/koding/db/mongodb/modelhelper

**Pull request merged**